examples/example-node-worker/main.ts

```
@@ -1,53 +1,177 @@
+        callback: onEntityUpdated,
+    });
+
+    console.log("Entities from worker", entities);
+    console.log(positionCount);
 
-process.on("SIGINT", () => {
-    // NOTE: do not forget to free sub here;
-    sub.free();
-    process.exit(0);
+    return [];
+    return [sub];
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)